cypress/locators/grid/locators.js

```diff
@@ -0,0 +1,3 @@
+// component preview locators
+export const POD = '[data-component="pod"]';
```

Comment:
do we need those locators ? I think we already have pod in `POD` component locators and `ROOT` in common, could you check?